common.blocks/syntax-highlighter/syntax-highlighter.js

```diff
@@ -0,0 +1,24 @@
+/* borschik:include:../../node_modules/prismjs/prism.js */
```

Comment:
this part can be wrapped not to expose `Prism` to global scope

Reply:
I am thinking here about moving this to block prism and use it as a dependency or just wrap it in syntax-highlighter module?